locales/en.yml

```diff
@@ -20,8 +20,137 @@ posts:
   add: Add post
   title: Feed
 profiles:
+  username: Username
```

Comment:
I would put it inside `profile` (singular), cuz I used `profiles` section for Community, i.e. profile list.